### ISSUE TITLE
Consider subscribing to shadowserver for reports about our networks

### ISSUE THREAD

**Issue by naved001** [CCI-MOC/ops-issues#1403]
Shadowserver can monitor networks for free and send daily/weekly/monthly reports about any security vulnerabilities on hosts on a network. CSAIL will usually forward us reports from shadowserver but we don't get any notifications like this from the NEU network (129.10.5.0/24) or the BU network (192.12.185.0/24). 

I think we should consider subscribing to these alerts for those networks. I am not sure if there are any downsides besides giving them explicit permission to scan hosts on our network.

https://www.shadowserver.org/what-we-do/network-reporting/get-reports/

**Comment by msdisme** [comment]
@hakasapl @larsks

**Comment by hakasapl** [comment]
My understanding is that this run externally from our infrastructure right? If so then I have no concerns, and I think we should buy it. If we have to run a daemon for it ourselves it would be good if they had support for the netgate firewall (pfsense pro)

**Comment by naved001** [comment]
@hakasapl yes, it's externally run and it's free so no need to pay for it.

**Comment by hakasapl** [comment]
Oh, it's free? That's great

**Comment by naved001** [comment]
I sent an email to NEU folks and see if they already have any network security reports and if they are okay with us subscribing to shadowserver.
